CHANGELOG.md

```diff
@@ -1,11 +1,10 @@
-x.x.x Release notes (yyyy-MM-dd)
+5.0.2 Release notes (2020-3-21)
 =============================================================
 ### Enhancements
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed a bug in Realm JS v5.0.0 and v5.0.1 in how the destruction of global objects was handled. This could lead to segfaults on Node.js version 12 and 13. (#2759)
```

Comment:
@blagoev @kneth Please keep changelogs focused on the user experience versus implementation.
In this case: When would these segfaults occur? What's an example of how the error would be visible?